packages/shell-api/src/shard.spec.ts

```diff
@@ -1210,8 +1210,11 @@ describe('Shard', () => {
         expect((await sh.startBalancer()).ok).to.equal(1);
         expect((await sh.isBalancerRunning()).mode).to.equal('full');
       });
-      it('reports state for collection', async() => {
-        expect(Object.keys(await sh.balancerCollectionStatus(ns))).to.include('balancerCompliant');
+      describe('balancerCollectionStatus', () => {
+        skipIfServerVersion(mongos, '< 4.4');
```

Comment:
is `mongos` a string here?

Reply:
It’s the reference to the test server (i.e. a `MongodSetup` from `integration-testing-hooks.ts`)